### PR TITLE
Cut down noise of plugins

### PR DESCRIPTION
**Pull request by julianoes**
This is an attempt to cut down on some of the noise by the plugins. Please check the commit messages for more details.

### CODE REVIEW

**Review by LorenzMeier — APPROVED** [review]

**Comment by julianoes** [comment]
We can do more though, especially for the models with camera like `typhoon_h480`. It spits out a bunch of warnings and errors that I mostly don't understand.

File: src/gazebo_gps_plugin.cpp [hunk 1]
```diff
@@ -86,34 +86,34 @@ void GpsPlugin::Load(physics::ModelPtr _model, sdf::ElementPtr _sdf)
 
   if (env_lat) {
     lat_home = std::stod(env_lat) * M_PI / 180.0;
+    gzmsg << "Home latitude is set to " << lat_home << ".\n";
```
Comment (TSC21) [inline src/gazebo_gps_plugin.cpp:89]: Why was this moved here?

File: src/gazebo_mavlink_interface.cpp [hunk 2]
```diff
@@ -208,21 +208,6 @@ void GazeboMavlinkInterface::Load(physics::ModelPtr _model, sdf::ElementPtr _sdf
           {
             std::string joint_name = channel->Get<std::string>("joint_name");
             joints_[index] = model_->GetJoint(joint_name);
-            if (joints_[index] == nullptr)
-            {
-              gzwarn << "joint [" << joint_name << "] not found for channel["
```
Comment (TSC21) [inline src/gazebo_mavlink_interface.cpp:213]: I would keep this as a gzdbg.

File: src/gazebo_motor_model.cpp [hunk 3]
```diff
@@ -33,7 +33,8 @@ void GazeboMotorModel::InitializeParams() {}
 
 void GazeboMotorModel::Publish() {
   turning_velocity_msg_.set_data(joint_->GetVelocity(0));
-  motor_velocity_pub_->Publish(turning_velocity_msg_);
+  // FIXME: Commented out to prevent warnings about queue limit reached.
+  // motor_velocity_pub_->Publish(turning_velocity_msg_);
```
Comment (TSC21) [inline src/gazebo_motor_model.cpp:37]: I don't actually think we need this at all. Better just remove it.

File: src/gazebo_gps_plugin.cpp (hunk 1 above)
Comment (julianoes) [reply src/gazebo_gps_plugin.cpp:89]: Because I wouldn't print the lat/lon if it's just the default.

File: src/gazebo_mavlink_interface.cpp (hunk 2 above)
Comment (julianoes) [reply src/gazebo_mavlink_interface.cpp:213]: Why? Is it an error? Or just for development?

File: src/gazebo_motor_model.cpp (hunk 3 above)
Comment (julianoes) [reply src/gazebo_motor_model.cpp:37]: Ok, I have no idea...

File: src/gazebo_gps_plugin.cpp (hunk 1 above)
Comment (Jaeyoung-Lim) [reply src/gazebo_gps_plugin.cpp:89]: If the world_has_origin is true, it still is not default. Could we add another print out there?

File: src/gazebo_motor_model.cpp (hunk 3 above)
Comment (Jaeyoung-Lim) [reply src/gazebo_motor_model.cpp:37]: I think this is used by the motor_failure_plugin. It was on the todo list of mine to remove the ROS dependency...but also fine with removing it

File: src/gazebo_wind_plugin.cpp [hunk 4]
```diff
@@ -142,7 +143,8 @@ void GazeboWindPlugin::OnUpdate(const common::UpdateInfo& _info) {
   wind_msg.set_time_usec(now.Double() * 1e6);
   wind_msg.set_allocated_force(force);
 
-  wind_pub_->Publish(wind_msg);
+  // FIXME: Commented out to prevent warnings about queue limit reached.
+  //wind_pub_->Publish(wind_msg);
```
Comment (Jaeyoung-Lim) [inline src/gazebo_wind_plugin.cpp:147]: We would need to add this back when we get https://github.com/PX4/sitl_gazebo/pull/375 in. but currently it isn't used anywhere

File: src/gazebo_gps_plugin.cpp (hunk 1 above)
Comment (julianoes) [reply src/gazebo_gps_plugin.cpp:89]: Ah ok, done in 314de47.

File: src/gazebo_motor_model.cpp (hunk 3 above)
Comment (julianoes) [reply src/gazebo_motor_model.cpp:37]: Ok, I guess let's have it commented out for now.

File: src/gazebo_wind_plugin.cpp (hunk 4 above)
Comment (julianoes) [reply src/gazebo_wind_plugin.cpp:147]: Ok thx.

**Comment by julianoes** [comment]
@Jaeyoung-Lim if you get a chance it would be good if we could do similar cleanup for the camera models. Should we add it here or do it in a later PR?

File: src/gazebo_mavlink_interface.cpp (hunk 2 above)
Comment (TSC21) [reply src/gazebo_mavlink_interface.cpp:213]: Development

**Comment by julianoes** [comment]
Ok, merging this for now. We can improve it later.

**Comment by Jaeyoung-Lim** [comment]
@julianoes Ah, sorry I missed this. I will cut down the camera model outputs
